src/demo-app/a11y/chips/chips-a11y.html

```diff
@@ -0,0 +1,54 @@
+<section>
+  <h2>Basic chips</h2>
+  <md-chip-list>
+    <md-chip>Chip 1</md-chip>
+    <md-chip>Chip 2</md-chip>
+    <md-chip>Chip 3</md-chip>
+  </md-chip-list>
+
+</section>
+
+<section>
+  <h2>Unstyled chips</h2>
+
+  <md-chip-list>
+    <md-basic-chip>Basic Chip 1</md-basic-chip>
+    <md-basic-chip>Basic Chip 2</md-basic-chip>
+    <md-basic-chip>Basic Chip 3</md-basic-chip>
+  </md-chip-list>
+</section>
+
+<section>
+  <h2>Removable chips in input container</h2>
+
+  <md-input-container>
+    <md-chip-list mdPrefix #chipList>
+      <md-chip *ngFor="let person of people" [color]="color">
+        {{person.name}}
+      </md-chip>
+    </md-chip-list>
+    <input mdInput placeholder="New Contributor..."
+           [mdChipInputFor]="chipList"
+           [mdChipInputAddOnBlur]="addOnBlur"
+           (mdChipInputTokenEnd)="add($event)" />
+  </md-input-container>
+</section>
+
+<section>
+  <h2>Colored chips</h2>
+  <md-chip-list>
+    <md-chip color="primary" selected="true">Primary</md-chip>
+    <md-chip color="accent" selected="true">Accent</md-chip>
+    <md-chip color="warn" selected="true">Warn</md-chip>
+  </md-chip-list>
+</section>
```

Comment:
Maybe add a comment here that this example is good to have for contrast-radio checking